scripts/histmakers/w_z_gen_dists.py

```diff
@@ -48,18 +48,51 @@
     0, 1, name="chargeVgen", underflow=False, overflow=False
 )
 
-qcdScaleByHelicity_Zhelper = wremnants.makeQCDScaleByHelicityHelper(is_w_like = True)
-qcdScaleByHelicity_Whelper = wremnants.makeQCDScaleByHelicityHelper()
-
-print("Did we fail yet?")
-wprocs = ["WplusmunuPostVFP", "WminusmunuPostVFP", "WminustaunuPostVFP", "WplustaunuPostVFP"]
-zprocs = ["ZmumuPostVFP", "ZtautauPostVFP"]
+axis_l_eta_gen = hist.axis.Regular(48, -2.4, 2.4, name = "prefsr_lepton_eta_gen")
+axis_l_pt_gen = hist.axis.Regular(29, 26., 55., name = "prefsr_lepton_pt_gen")
+axes_l_gen = [axis_l_eta_gen, axis_l_pt_gen]
+
+wprocs_bugged = [
+    "WplusmunuPostVFP", 
+    "WminusmunuPostVFP",
+    "WminustaunuPostVFP",
+    "WplustaunuPostVFP"
+]
+wprocs_bugfix = [
+    "WplusmunuPostVFP_bugfix", 
+    "WminusmunuPostVFP_bugfix",
+    "WminusmunuPostVFP_bugfix_slc7",
+    "WplusmunuPostVFP_bugfix_slc7",
+    "WplusmunuPostVFP_bugfix_h2",
+]
+wprocs = [*wprocs_bugged, *wprocs_bugfix]
+wprocs_bugged_to_check = [ # mu only for making comparison plots between bugged and bugfix samples
+    "WplusmunuPostVFP", 
+    "WminusmunuPostVFP",
+]
+zprocs_bugged = ["ZmumuPostVFP", "ZtautauPostVFP"]
+zprocs_bugfix = ["ZmumuPostVFP_bugfix", "ZmumuPostVFP_bugfix_slc7"]
+zprocs = [*zprocs_bugged, *zprocs_bugfix]
+zprocs_bugged_to_check = ["ZmumuPostVFP"]
+
+time_loading_finished = time.time()
+loading_time = time_loading_finished - time_start
+print("time it takes to load datasets: ", loading_time)
 
 
 def build_graph(df, dataset):
     print("build graph")
     print(dataset.name)
     results = []
+    
+    if dataset.is_data:
```

Comment:
I guess this doesn't hurt anything but not sure we ever want to run the data

Reply:
Indeed, now data is not loaded by default:
```parser.add_argument("--filterProcs", type=str, nargs="*", help="Only run over processes matched by (subset) of name", default=["Wplus", "Wminus", "Zmumu", "Ztautau"])```
I guess we can keep them here in case we will need it later. Probably need to refactor this a little bit in later iterations so the logic is more clear. But I think it's fine for this PR.